docs/content/reference/cli.md

```diff
@@ -9,56 +9,100 @@ The Boltzmann CLI creates and updates a node project for you, with a boltzmann.j
 
 ```shell
-hello|⇒ npx boltzmann-cli --redis=on .
-npx: installed 1 in 1.025s
-Scaffolding a Boltzmann service in /Users/ceejbot/code/hello/.
-    loaded settings from existing package.json
+code|⇒ npx boltzmann-cli hello --redis=on
+npx: installed 1 in 2.659s
+Scaffolding a Boltzmann service in /Users/cj/code/hello
+    initializing a new NPM package...
     writing boltzmann files...
     updating dependencies...
+        adding are-we-dev @ ^1.0.0
+        adding culture-ships @ ^1.0.0
+        adding find-my-way @ ^2.2.1
+        adding bole @ ^4.0.0
+        adding ajv @ ^6.12.2
+        adding dotenv @ ^8.2.0
+        adding accepts @ ^1.3.7
         adding handy-redis @ ^1.8.1 (redis activated)
         adding redis @ ^3.0.2 (redis activated)
+        adding cookie @ ^0.4.1
+        adding tap @ ^14.10.7 (dev)
+        adding @hapi/shot @ ^4.1.2 (dev)
+        adding bistre @ ^1.0.1 (dev)
+        adding eslint @ ^7.1.0 (dev)
+        adding @typescript-eslint/parser @ ^3.1.0 (dev)
+        adding @typescript-eslint/eslint-plugin @ ^3.1.0 (dev)
+        adding eslint-config-prettier @ ^6.11.0 (dev)
+        adding eslint-plugin-prettier @ ^3.1.3 (dev)
+        adding prettier @ ^2.0.5 (dev)
     writing updated package.json...
     running package install...
-Boltzmann at 0.1.1 with redis, githubci, status, ping
+Boltzmann at 0.1.2 with redis, githubci, status, ping
 ```
 
 Pass `--feature=off` to remove a feature you have previously enabled. For example, if you decide you don't need redis after all:
 
 ```shell
-hello|⇒ npx boltzmann-cli --redis=off .
-npx: installed 1 in 1.012s
-Scaffolding a Boltzmann service in /Users/ceejbot/code/hello/.
+code|⇒ npx boltzmann-cli hello --redis=off
+npx: installed 1 in 0.809s
+Scaffolding a Boltzmann service in /Users/cj/code/hello
     loaded settings from existing package.json
     writing boltzmann files...
     updating dependencies...
         removing handy-redis (redis deactivated)
         removing redis (redis deactivated)
     writing updated package.json...
     running package install...
-Boltzmann at 0.1.1 with githubci, status, ping
+Boltzmann at 0.1.2 with githubci, status, ping
 ```
 
+Boltzmann will manage its dependencies when a feature is flipped on or off, but will not make other changes to an existing package.json. You are free to update its dependencies as you see best.
+
+## Command-line flags
+
+Boolean options:
+
+- `all`: Enables all features. Added in version 0.1.3.
+- `website`: Enable website feature set (templates, csrf). Added in version 0.1.3.
+- `selftest`: Turns on features for testing Boltzmann while developing it.
+- `docs`: Open the Boltzmann documentation in a web browser. Added in version 0.1.3.
+- `force`: Update a git-repo destination even if there are changes.
+- `silent`: Suppress all output except errors.
+- `verbose`: Log even more. Pass -v or -vv to increase verbosity.
+- `version`: Print version information.
```

Comment:
We may wish to turn flags and feature flippers into h3 headings so they can be deep-linked to (both by outsiders as well as by us elsewhere in the documentation.)